app/assets/javascripts/application.js

```diff
@@ -236,8 +236,30 @@ $(document).on("turbolinks:load", function() {
           id: "genres",
           label: "Genres",
           type: "string",
-          input: "select",
-          operators: ["in"],
+          // input: "select",
+          input: function(rule, name) {
```

Comment:
I left a ton of comments. Hopefully that's useful. This seemed to be one of the best ways to solve this problem. First time it loads it uses the return value from `input` method. Subsequent loads we have to call `input()` manually from the event handler that's called when the operator changes.